components/input/src/input-field/input-field.js

```diff
@@ -90,6 +97,10 @@ const InputFieldProps = {
     /** The [native `autocomplete` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-autocomplete) */
     autoComplete: PropTypes.string,
     className: PropTypes.string,
+    /** function to clear the input value */
+    clearText: PropTypes.func,
```

Comment:
@d-rita do we need a separate callback function for this?  I think it might be possible to just call `setValue('')`?  Maybe there are cases when you want to reset it to the default value... but that could be handled by the setValue callback?

Reply:
In the Input component, I could use the `onChange` handler that we pass in to update value to an empty string. Then within the parent component where this `InputField` is being used, the `onChange` handler (which uses the setValue) can update the value accordingly. If there is a defaultValue, that can also be set within there.